File: src/main/java/um/haberes/rest/model/AntiguedadLimite.java
/**
 * 
 */
package um.haberes.rest.model;

import java.io.Serializable;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * @author daniel
 *
 */
@Entity
@EqualsAndHashCode(callSuper = false)
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class AntiguedadLimite extends Auditable implements Serializable {
	/**
	* 
	*/
	private static final long serialVersionUID = -5174803627592479795L;

	@Id
	private Integer desde = 0;

	private Integer hasta = 0;
	private Integer porcentaje = 0;

}

File: src/main/java/um/haberes/rest/kotlin/model/AntiguedadLimite.kt
package um.haberes.rest.kotlin.model

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table
data class AntiguedadLimite(

    @Id
    var desde: Int = 0,

    var hasta: Int = 0,
    var porcentaje: Int = 0

) : Auditable()
